source/data-formats/time-series.txt

```diff
@@ -0,0 +1,204 @@
+.. _pymongo-time-series:
+
+================
+Time Series Data
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use {+driver-short+} to store
+and interact with **time series data**.
+
+Time series data is composed of the following components:
+
+- Measured quantity
+- Timestamp for the measurement
+- Metadata that describes the measurement
+
+The following table describes sample situations for which you could store time
+series data:
+
+.. list-table::
+   :widths: 33, 33, 33
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Situation
+     - Measured Quantity
+     - Metadata
+
+   * - Recording monthly sales by industry
+     - Revenue in USD
+     - Company, country
+
+   * - Tracking weather changes
+     - Precipitation level
+     - Location, sensor type
+
+   * - Recording fluctuations in housing prices
+     - Monthly rent price
+     - Location, currency
+
+.. _pymongo-time-series-create:
+
+Create a Time Series Collection
+-------------------------------
+
+.. important:: Server Version for Time Series Collections
+
+   To create and interact with time series collections, you must be
+   connected to a deployment running {+mdb-server+} 5.0 or later.
+
+You can create a time series collection to store time series data. To create
+a time series collection, pass the following arguments to the ``create_collection()``
+method:
+
+- Name of the new collection to create
+- ``timeseries`` argument 
+
+The ``timeseries`` argument is of type ``dict``. It contains the following fields:
+
+- ``timeField``: Specifies the field that stores a timestamp in each time series
+  document.
+- ``metaField``: Specifies the field that stores metadata in each time series 
+  document.
+- ``granularity``: Specifies the approximate time between consecutive timestamps.
+  The possible values are ``'seconds'``, ``'minutes'``, and ``'hours'``.
+- ``bucketMaxSpanSeconds``: Sets the maximum time between timestamps in the 
+  same bucket.
+- ``bucketRoundingSeconds``: Sets the number of seconds to round down by when 
+  MongoDB sets the minimum timestamp for a new bucket. Must be equal to 
+  ``bucketMaxSpanSeconds``.
+
+See :manual:`Command Fields </reference/command/create/#command-fields>`
+to learn more about these fields.
+
+Example
+~~~~~~~
+
+The following example creates a time series collection named ``october2024`` with the
+``timeField`` option set to ``"timestamp"``:
+
+.. code-block:: python
+
+    database = client.get_database("weather")
+
+    time_series_options = {
+        "timeField": "timestamp"
+    }
+
+    database.create_collection("october2024", timeseries=time_series_options)
+
+To check if you successfully created the collection, you can print out a list of
+all collections in the database:
```

Comment:
Printing every collection? This seems like overkill. This might be a natural place to look at the special indexes of time-series collections.  `indexes = collection.index_information()` would give you a dict with a key not only for `_id` but also `_ts_` and the latter would be a compound index on timeField and metaField

Reply:
As discussed offline, keeping `list_collections()` but filtering by collection name.